### PR TITLE
refactor(devices): update English docs for devices and products

### PR DESCRIPTION
**Pull request by oucb**

### CODE REVIEW

File: en/products/codec.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+### Running
+
+Click the `Running` button to check if the results are as expected.
+
+#### Running result description
+
+- Error: script error or validation error
+- Status_code: status code
+- Result: the result after processing which will be specified in the follows.
```
Comment (Zhiw) [inline en/products/codec.md]: Error -> error
Status_code -> status_code
Result -> result
You  can see the result below

File: en/products/codec.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+1. Log in with the super administrator account and go to: Device Management -> Product Management -> Codec Plugin review, click the `Rewiew` button of the corresponding product codec plugin.
+2. Check the 'pass' in the review result, then click `OK` to make the codec plugin take effect.
+
+
+## 3. Format description
+
+`data_type` and `data` are required to fill.
+
+### 1) Message report
```
Comment (Zhiw) [inline en/products/codec.md]: Message report -> Event (or Device Event)

File: en/products/codec.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+  "task_id": "d89ed1925a6d5c3184e1934244ad0000",
+  "data": {
+    "mode": "cold",
+    "other_para": "test"
+  }
+}
+```
+
+- data_type:`request`
+
```
Comment (Zhiw) [inline en/products/codec.md]: I think the blank line is unnecessary

Comment (oucb) [reply en/products/codec.md]: @Zhiw all resolved

**Pull request merged**